snippets/base/templates/base/includes/snippet_as.css

```diff
@@ -18,6 +19,14 @@
   }
 }
 
+#snippets-container a {
+  color: var(--newtab-link-primary-color);
+}
+
+.dark-theme #snippets-container a {
```

Comment:
Why change this?

Reply:
those links are bold in the dark theme. see spec: https://mozilla.invisionapp.com/share/H4FW3RYFXVK#/screens/279681634_New_Tab_Dark_Theme_Spec

Reply:
Do you have a spec for the normal theme?

Reply:
unfortunately, there is no up to date spec for the normal theme. It just evolved over time but the goal was to not change it when implementing the dark theme.